doc/pr-review-policy.md

```diff
@@ -0,0 +1,96 @@
+## Pull Request Review Policy
+
+This document details the pull request policy of the Status Mobile repository / team.
+
+- [Terms](#terms)
+- [Policy](#policy)
+  - [Overview](#overview)
+  - [Full Policy](#fully-policy)
+- [Rationale](#rationale)
+  - [Front pressure and Demand control](#front-pressure-and-demand-control)
+  - [Back pressure and Supply control](#back-pressure-and-supply-control)
+  - [Honour System](#honour-system)
+- [Review Support](#review-support)
+- [Background](#background)
+
+### Terms:
+
+| Term                 | Definition                                                                                                    |
+|----------------------|---------------------------------------------------------------------------------------------------------------|
+| **Code owner**       | The team member submitting a pull request.                                                                    |
```

Comment:
for me when I hear the term Code Owner I confuse it with Code Owners functionality in Github 
ref: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

Can we change it to something or anything else ?
I have no strong preference.

Reply:
Yes @siddarthkay that makes sense, I will change this to maybe "Requester"?